README.md

```diff
@@ -31,15 +31,23 @@ To add a new brand:
 
 # Releasing
 
+## Alpha for release testing
+
+1. Create a git tag starting with `v` and ending with the alpha version, e.g. `v1.2.3-alpha.1` and push the tag to GitHub.
```

Comment:
I assume it's cool to tag on a branch vs master? Either from a CI perspective or from a best practices perspective ;)

Reply:
In cht-core we get the change passed code review, then AT, then merge to `master` which is eventually tagged for release testing. I'd recommend the same here, that we should tag `master` not any old feature branch. What do you think? Should I add this to this doc do you think?

Reply:
Yeah - Makes total sense to mirror our cht-core process here. Good thinking!

Adding just a sentence about this is good enough - maybe with a link? "Be sure the release has passed code review, been through AT and is merged to `master`, similar to the [CHT code workflow](https://docs.communityhealthtoolkit.org/contribute/code/workflow/)."

Link may be more confusing though as we don't follow the exact. same process.